monitoring/daily_reporter/src/ingests.py

```diff
@@ -72,3 +99,116 @@ def get_dev_status(ingest):
             return "stalled"
         else:
             return "processing"
+
+
+def is_known_failure(ingest_id):
+    return ingest_id in {
+        # digitised/b12948421, failed with an unknown unpacker error
+        # Reingested as f9850ea4-28b3-4ef2-a385-d0227fb5167a
+        "dda879ab-ba7b-4e87-a34b-6511e6e1457b",
```

Comment:
I'm not sure I like this ... it's an infinitely growing list, that relies on a dev properly commenting things to make it sensible.